src/crimson/os/seastore/transaction.h

```diff
@@ -116,6 +116,18 @@ class Transaction {
     return weak;
   }
 
+  void add_rbm_allocated_blocks(rbm_extent_t &extent) {
```

Comment:
I need a concise, clear explanation of how this is meant to fit into TransactionManager and Cache.